src/email_service/validate_input/lambda_function.py

```diff
@@ -28,22 +29,33 @@
 DEFAULT_DISPLAY_NAME = "AWS Educate 雲端大使"
 DEFAULT_REPLY_TO = "awseducate.cloudambassador@gmail.com"
 DEFAULT_SENDER_LOCAL_PART = "cloudambassador"
+DEFAULT_RECIPIENT_SOURCE = "SPREADSHEET"
+EMAIL_PATTERN = r"[^@]+@[^@]+\.[^@]+"
 
-# Initialize AWS SQS client
+# Initialize AWS clients and repositories
 sqs_client = boto3.client("sqs")
-
-# Initialize RunRepository
 run_repository = RunRepository()
 
 
-def get_file_info(file_id, access_token):
-    """
-    Retrieve file information from the file service API.
+def create_error_response(status_code: int, message: str) -> dict[str, Any]:
```

Comment:
Great way to handle all the erorr responses with a `create_error_response` function !